src/app/personside/infotabs/oppfolging/__snapshots__/OppfolgingContainer.test.tsx.snap

```diff
@@ -112,27 +112,28 @@ exports[`Viser oppfølgingcontainer med alt innhold 1`] = `
   background-color: white;
 }
 
-.c11 table {
+.c11 [role='table'] {
```

Comment:
Ser at tabell-endringene treffer ganske bredt, men det er testet og ser/funker likt?

Reply:
Jupp tatt en runde rundt i modia for å sjekke at andre tabellene ikke er påvirket.